pkg/command/checks/command.go

```diff
@@ -36,13 +36,24 @@ func renderPlain(checks []config.Check, out io.Writer) error { //nolint:unparam
 }
 
 const (
-	markDownTemplateStr = `# KubeLinter checks
+	markDownTemplateStr = `# KubeLinter cheks
 
 KubeLinter includes the following built-in checks:
 
-| Name | Enabled by default | Description | Remediation | Template | Parameters |
-| ---- | ------------------ | ----------- | ----------- | -------- | ---------- |
-{{ range . }} | {{ .Check.Name}} | {{ if .Default }}Yes{{ else }}No{{ end }} | {{.Check.Description}} | {{.Check.Remediation}} | {{.Check.Template}} | {{ mustToJson (default (dict) .Check.Params ) | codeSnippetInTable }} |
+{{ range . -}}
+## {{ .Check.Name}}
+
+**Enabled by default**: {{ if .Default }}Yes{{ else }}No{{ end }}
+
+**Description**: {{.Check.Description}}
+
+**Remediation**: {{.Check.Remediation}}
+
+**Template**: [{{.Check.Template}}](generated/templates.md#{{.Check.Template}})
```

Comment:
Let's add another field to `augmentedCheck` below called `TemplateLink`, and populate it with the link to the template. The link can be derived as follows:

1. Call [`templates.Get`](https://github.com/stackrox/kube-linter/blob/main/pkg/templates/registry.go#L24) with `chk.Template` for each check. This will return the full template.
1.  Do `strings.Join(strings.Fields(strings.ToLower(template.HumanName)), "-")` to derive the TemplateLink
1. Use template link here (should look like the suggestion below)

```suggestion
**Template**: [{{.Check.Template}}](generated/templates.md#{{.TemplateLink}})
```

Reply:
Thanks @viswajithiii for explanation, I tried to use this but couldn't  fix it. Can you fix this one?

Reply:
Sure!